src/Domain.LinnApps/Parts/MechPartSource.cs

```diff
@@ -161,5 +161,55 @@ public class MechPartSource
         public string FootprintRef { get; set; }
 
         public string RkmCode { get; set; }
+
+        public ICollection<MechPartPurchasingQuote> PurchasingQuotes { get; set; }
```

Comment:
Why are you using ICollection instead of IEnumerable?

Reply:
I need to add, remove and update elements in the these lists. I don't think IEnumerable lets you do that? Pretty sure I got errors when these were IEnumerables so just switched to ICollection